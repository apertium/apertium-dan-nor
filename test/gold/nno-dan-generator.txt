[4IAP1Qxnrab0]
Så hun ikke ordene? [/option]
[/4IAP1Qxnrab0]
[6oqVIKDxYJzh]
den trætte mand [/option]
[/6oqVIKDxYJzh]
[SPQofzn-jOR7]
    Wikipedia er en international internetbaseret encyklopædi der udgives af den ideelle organisation Wikimedia Foundation [/option]
[/SPQofzn-jOR7]
[kMFdz2mKdYmx]
Aasen oversatte også digt; hans forlag omsatte for 4 milliarder årlig [/option]
[/kMFdz2mKdYmx]
[zZNykk8ceqd7]
hun kan ikke lide dette noget særligt [/option]
[/zZNykk8ceqd7]
