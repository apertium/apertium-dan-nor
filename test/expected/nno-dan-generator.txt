[4IAP1Qxnrab0#0] Så hun ikke ordene?
[/4IAP1Qxnrab0]
[6oqVIKDxYJzh#0] den trætte mand
[/6oqVIKDxYJzh]
[SPQofzn-jOR7#0] Wikipedia er en international internetbaseret encyklopædi, som bliver udgivet af den ideelle organisation Wikimedia Foundation
[/SPQofzn-jOR7]
[kMFdz2mKdYmx#0] Aasen oversatte også digt; hans forlag omsatte for 4 milliarder årlig
[/kMFdz2mKdYmx]
[zZNykk8ceqd7#0] hun kan ikke lide dette noget særligt
[/zZNykk8ceqd7]
